ramp4-pcar4/story-ramp
# add background image to introduction slide

### Related Item(s)
https://github.com/ramp4-pcar4/tcei-tmx-cwa-storylines/issues/24

### Changes
- Adds a `backgroundImage` property to the introduction slide config schema.
- Adds `titleColour`, `subtitleColour`, and `buttonColour` to customize the other elements in the intro slide.
- Fixes the background image transition animation when using the WET templates.

### Testing
Steps:
1. Open the demo page.
2. Ensure that the background image is displaying properly on different screen sizes.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/story-ramp/472)
